staging/src/k8s.io/api/storage/v1beta1/types.go

```diff
@@ -291,10 +291,41 @@ type CSIDriverSpec struct {
 	// "csi.storage.k8s.io/pod.name": pod.Name
 	// "csi.storage.k8s.io/pod.namespace": pod.Namespace
 	// "csi.storage.k8s.io/pod.uid": string(pod.UID)
+	// "csi.storage.k8s.io/ephemeral": "true" iff the volume is an ephemeral inline volumed
+	//                                 defined by a CSIVolumeSource, otherwise "false"
```

Comment:
Because this is new, you also have to define the behavior of absence.  So for example,

"If this annotation is not present, the CSI implementation must assume the cluster does not support ephemeral inline volume sources, and that this volume must be persistent."

Reply:
Doesn't have to be inline to the annotation, you could move it to a separate paragraph after the annotations are listed.

Reply:
Added this paragraph:

"csi.storage.k8s.io/ephemeral" is a new feature in Kubernetes 1.16. It is only
required for drivers which support both modes. Other drivers can leave pod info
disabled and/or ignore this field. As Kubernetes 1.15 doesn't support this field,
drivers can only support one mode when deployed on such a cluster and the
deployment determines which mode that is, for example via a command line parameter
of the driver.